onmt/modules/Decoder.lua

```diff
@@ -226,7 +234,7 @@ Returns:
  1. `out` - Top-layer hidden state.
  2. `states` - All states.
 --]]
-function Decoder:forwardOne(input, prevStates, context, prevOut, t)
+function Decoder:forwardOne(input, _, prevStates, context, prevOut, t)
```

Comment:
I know it is for preparing the coverage attention, but ideally this change should not be part of this PR. Same for `initializeSpecialStates` and `cloneId`.